api-reference/receipts/endpoints.markdown

```diff
@@ -0,0 +1,672 @@
+  "self": "https://us.api.concursolutions.com/receipts/v4/85b76a2bf51a4ed7b8b252994d7d9e2b",
+  "template": "https://us.api.concursolutions.com/receipts/v4/{receiptId}"
+}
+```
+
+[Back to Top](#endpoints)
+
+##### Endpoint: Get Receipts By UserId
+
+###### _GET /v4/users/:userId_
+
+|Parameter|Requirement|Value|
+|---|---|---|
+|userId|required|The UUID of the user whose receipts will be returned.|
+
+Returns all receipts for a given user ID.
+
+_Example Requests:_
+
+cURL:
+
+```shell
+curl -H "Authorization: Bearer {YOUR ACCESS TOKEN}" https://us.api.concursolutions.com/receipts/v4/users/{USER ID}
+```
+
+HTTPie:
+
+```shell
+http https://us.api.concursolutions.com/receipts/v4/users/{USER ID} "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+_Example Response:_
+
+```json
+{
+  "receipts": [
+    {
+      "dateTimeReceived": "2016-09-28T21:41:21.087Z",
+      "id": "85b76a2bf51a4ed7b8b252994d7d9e2b",
+      "image": "",
+      "imageId": "",
+      "receipt": {
+        ...Receipt JSON...
+      },
+      "userId": "08bdda1e-0d4f-4261-9f1b-f9b8d9f817d6",
+      "validationSchema": "http://schema.concursolutions.com/car-rental-receipt.schema.json",
+      "self": "https://us.api.concursolutions.com/receipts/v4/85b76a2bf51a4ed7b8b252994d7d9e2b",
+      "template": "https://us.api.concursolutions.com/receipts/v4/{receiptId}"
+    },
+    {
+      "dateTimeReceived": "2016-09-28T19:59:30.488Z",
+      "id": "df8c1161d917439c9e6f141fd0d6b588",
+      "image": "",
+      "imageId": "",
+      "receipt": {
+        ...Receipt JSON...
+      },
+      "userId": "08bdda1e-0d4f-4261-9f1b-f9b8d9f817d6",
+      "validationSchema": "http://schema.concursolutions.com/car-rental-receipt.schema.json",
+      "self": "https://us.api.concursolutions.com/receipts/v4/df8c1161d917439c9e6f141fd0d6b588",
+      "template": "https://us.api.concursolutions.com/receipts/v4/{receiptId}"
+    },
+    ...
+  ]
+}
+```
+
+[Back to Top](#endpoints)
+
+##### Endpoint: Get Receipt Image
+
+###### _GET /v4/:receiptId/image_
+
+|Parameter|Requirement|Value|
+|---|---|---|
+|receiptId|required|The UUID of the receipt associated with the image.|
+
+If an image or PDF document was generated by or POSTed to Receipts v4, this endpoint can return the image in the same format that it was originally received by the API. Images for receipts created with v3 of the API are _not_ accessible via this endpoint.
+
+_Example Requests:_
+
+cURL:
+
+```shell
+curl -H "Authorization: Bearer {YOUR ACCESS TOKEN}" https://us.api.concursolutions.com/receipts/v4/{RECEIPT ID}/image
+```
+
+HTTPie:
+
+```shell
+http https://us.api.concursolutions.com/receipts/v4/{RECEIPT ID}/image "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+[Back to Top](#endpoints)
+
+### Image-Only Receipts
+
+|Endpoint|Response Format|Request Summary|
+|---|---|---|
+|[POST /v4/users/:userId/images](#endpoint-post-an-image-only-receipt)|N/A|Post a receipt image|
```

Comment:
I think these urls and nomenclature have to be updated as well, to `image-only-receipts`.

Reply:
✅